python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -47,7 +48,19 @@ def __init__(self, port):
             self.port = port
             self.ip = '127.0.0.1'
             self.use_cdn = True
-            ExplanationDashboard.DashboardService._local_port_available(self.ip, self.port)
+            if self.port is None:
```

Comment:
Yes, I'd thought about copying in Sam & Harsha's free-port function (it samples 10 random ports in a larger window and fails if all 10 are occupied). But I wasn't exactly clear on how it worked and didn't want to increase the footprint of the change more than necessary. Thank you for adding this, and for making clear to the user that they can always specify a port in the args.